### PR TITLE
update Have#respond_to signature to quiet warning

### PR DESCRIPTION
**Pull request by mwhipple**

### CODE REVIEW

**Comment by JonRowe** [comment]
Hi, thanks, could you rebase off master to get a working build please :)

File: lib/rspec/collection_matchers/have.rb [hunk 1]
```diff
@@ -116,8 +116,8 @@ def description
         "have #{relative_expectation} #{@collection_name}"
       end
 
-      def respond_to?(m)
-        @expected.respond_to?(m) || super
+      def respond_to?(m, include_all: false)
```
Comment (JonRowe) [inline lib/rspec/collection_matchers/have.rb]: Ah, I've just realised you've used a keyword argument here, that's not going to work on all versions of Ruby that this supports... maybe try using a splat and just passing through the arguments, or is the official definition with keyword args? If so you'll have to define multiple versions I suspect.

**Comment by mwhipple** [comment]
yeah, actually making sure the code works is going to take a little more
effort :/. I'll get everything green shortly.

**Comment by JonRowe** [comment]
Thanks for taking the time to tackle this!

**Comment by mwhipple** [comment]
This should be all set now. I'm not sure why I used the keyword arg: the standard implementation is the positional with default which is this has been updated to match.

**Comment by JonRowe** [comment]
Thanks!

**Pull request merged**